hstest/testing/plotting/drawing.py

```diff
@@ -0,0 +1,5 @@
+class Drawing:
```

Comment:
It would be great to have list of supported libraries and plot types in some class like

```python
class DrawingLibrary:
    matplotlib = "matplotlib"
    pandas = "pandas"
    seaborn = "seaborn"
``` 
and
```python
class DrawingType:
    scatter = "scatter"
    bar = "bar"
    violin = "violin"
    ...
```

And guarantee that values in `Drawing` objects are the same values as in these classes.